alf/algorithms/actor_critic_algorithm.py

```diff
@@ -95,8 +96,7 @@ def __init__(self,
             name (str): Name of this algorithm.
         """
         if epsilon_greedy is None:
-            epsilon_greedy = alf.get_config_value(
-                'TrainerConfig.epsilon_greedy')
+            epsilon_greedy = alf.utils.common.get_epsilon_greedy(config)
```

Comment:
It feels strange to have a function in common just for getting epsilon_greedy.
epsilon_greedy is algorithm specific. It's better to set as the argument of the algorithm.